nixos/modules/services/networking/networkmanager.nix

```diff
@@ -448,6 +448,10 @@ in {
     };
 
     systemd.services.ModemManager.aliases = [ "dbus-org.freedesktop.ModemManager1.service" ];
+    systemd.services.ModemManager.serviceConfig.ExecStart = [
+      ""
+      "${pkgs.modemmanager}/sbin/ModemManager --filter-policy=STRICT"
```

Comment:
Can you add a comment here that we're overriding the upstream-provided unit file, and link to the issue (so we know when it's become a default and we can remove our local hack)

Reply:
@flokli sure.